app/models/movie.rb

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
 
 class Movie < ApplicationRecord
-  attribute :duration, :string
-  has_many :screenings
+  has_many :screenings, dependent: :destroy
   has_many :halls, through: :screenings
-  validates :title, :duration, presence: true
+  validates_associated :screenings, :halls
+  validates :title, :description, :duration, presence: true
+  validates :title, length: { maximum: 255, too_long: '%<count>s characters is the maximum allowed' }
```

Comment:
`string` type in postgresql is `varchar(255)` so it already has a limit on database level, but it's a good place to customize error message